internal/api/v1alpha1/authproxyworkload_webhook.go

```diff
@@ -196,3 +199,53 @@ func validateWorkload(spec *WorkloadSelectorSpec, f *field.Path) field.ErrorList
 
 	return errs
 }
+
+func validateInstances(spec *[]InstanceSpec, f *field.Path) field.ErrorList {
```

Comment:
Nit: Go doc is your fiend and makes it easy to get into this function without requiring a thorough reading. It's fresh now, but in a few months, no one will remember what "validate" means unless they read all the source code.

Reply:
Good point. I wrote docs for the validate methods.